xtime/range.go

```diff
@@ -0,0 +1,34 @@
+// Package xtime extends Go's time. So far it contains an implementation of a time range.
+package xtime
+
+import "time"
+
+// Range defines a time range that contains time t if From <= t < To.
+type Range struct {
+	From, To time.Time
```

Comment:
Now...the eternal struggle in my mind...should this be from/to...or start/end ? Currently Im working with start/end on the enricher =P.

Reply:
When I think about ranges in a general purpose way it matches more with start/end

Reply:
In the ingestion we're mixing both :sweat_smile:

I don't have a preference, but we should probably agree on a standard.

Reply:
Not sure... :rofl: (sometimes I use from/to...but usually on the CLI because then the CLI itself is more specific and it seems to read better there)

Reply:
but on APIs I went with start/end..for the themes....hmmm. Im slightly more prone towards start/end.

Reply:
WDYT @guilherme-santos ?

Reply:
On search I also have a range, but it's more like a operator Greater, GreaterOrEqual and so on. But I do have some cases where I call Start/End for time, and From/To to other things (like id for instance).

Reply:
OK Ill vote for start/end :rofl: (but Im OK with from/to).

Reply:
I think for time start/end is quite nice! +1 for start/end!